src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageprisonvisitsorchestration/service/PrisonerProfileService.kt

```diff
@@ -69,4 +87,8 @@ class PrisonerProfileService(
       size = MAX_VISIT_RECORDS,
     )
   }
+
+  private fun getBlankPrisonerBasicInfo(prisonerId: String): PrisonerBasicInfoDto {
+    return PrisonerBasicInfoDto(prisonerId, null, null, null, null)
+  }
```

Comment:
don't understand this,... if it has not got the details we should not return them I think?

Reply:
updated to not return if not found.